api/src/test/java/org/openmrs/scheduler/SchedulerServiceTest.java

```diff
@@ -413,4 +414,22 @@ public void shouldSaveLastExecutionTime() throws Exception {
 		assertEquals("Last execution time in seconds is wrong", actualExecutionTime / 1000, td.getLastExecutionTime()
 		        .getTime() / 1000, 1);
 	}
+
+	/**
+	 * @see SchedulerService#getTaskByUuid(String)
+	 */
+	@Test
+	public void getTaskByUuid_shouldGetTaskByUuid() throws Exception {
+		TaskDefinition td = Context.getSchedulerService().getTaskByUuid("1365a6da-6493-4e9b-b950-5af1b392aaa3");
+		assertNotNull(td);
+	}
+
```

Comment:
I would also write some unhappy paths like, uuid is `null` and uuid is sql statement like `abc' or '1' = '1`

Reply:
i think if there is a brief code snippet which can strike the appoint,then its worth going for and avoid unnecessary code repetition